src/openforms/registrations/contrib/stuf_zds/plugin.py

```diff
@@ -365,3 +361,20 @@ def get_config_actions(self):
                 ),
             ),
         ]
+
+    def pre_register_submission(self, submission: "Submission", options: dict) -> None:
```

Comment:
I'd move this method to be above `register_submission` so that it reads a bit more naturally.